deploy/parametersStoreClient.js

```diff
@@ -0,0 +1,37 @@
+const AWS = require('aws-sdk');
+
+const getParametersByPath = function(path, awsRegion, nextToken = null) {
+  const ssm = new AWS.SSM({ region: awsRegion });
+  let params = { Path: path };
+  if (nextToken !== null) params.NextToken = nextToken;
+
+  return new Promise(function(resolve, reject) {
+    ssm.getParametersByPath(params, function(err, data) {
+      if (err) reject(err);
+      else {
+        let secrets = parse(data);
+        if (data.NextToken) {
+          getParametersByPath(path, awsRegion, data.NextToken).then(function(
+            nextSecrets
+          ) {
+            resolve(secrets.concat(nextSecrets));
+          });
+        } else {
+          resolve(secrets);
+        }
+      }
+    });
+  });
+};
+
+function parse(data) {
+  const secrets = [];
+  data.Parameters.forEach(function(secret) {
+    const name = secret.Name.replace(/^.*\//, '');
+    const value = secret.Value;
+    secrets.push([name, value]);
+  });
+  return secrets;
+}
+
+module.exports = getParametersByPath;
```

Comment:
@rodrei let me know if I'm missing something (I suspect I am), but why aren't we referencing the SSM params directly? eg ` DDB_TABLE: ${ssm:/path/to/dynamodbTable}`? https://serverless.com/framework/docs/providers/aws/guide/variables/#reference-variables-using-the-ssm-parameter-store